packages/babel-preset/index.js

```diff
@@ -30,14 +29,12 @@ module.exports = (
 ) => ({
   presets: [
     [
-      presetEnv,
+      'babel-preset-env-modules',
       {
         modules,
-        loose: true,
         ignoreBrowserslistConfig: true,
-        shippedProposals: true,
-        include: ['proposal-object-rest-spread'],
         targets: {
+          esmodules: true,
```

Comment:
i don't think this is needed? or at least it would override the browsers setting

Reply:
Ah you're right.  I copied it over from the defaults without reading what it actually did.  Fixing